application/src/main/java/org/togetherjava/tjbot/config/FreeCommandConfig.java

```diff
@@ -0,0 +1,67 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Config instance for the Free Command System see
+ * {@link org.togetherjava.tjbot.commands.free.FreeCommand}
+ *
+ * The Json looks as follows:
+ * 
+ * <pre>
+ * "freeCommand": [
+ *   {
+ *       "statusChannel": long_number,
+ *       "monitoredChannels": [long_number, long_number]
+ *   }]
+ * </pre>
+ * 
+ * Additional Guilds may add their settings by adding additional {"statusChannel....... }
+ *
+ * The long channel ID can be found by right-clicking on the channel and selecting 'Copy ID'
+ */
+@SuppressWarnings("ClassCanBeRecord")
+@JsonRootName(value = "freeCommand")
+public final class FreeCommandConfig {
+    public static final long INACTIVE_DURATION = 1;
+    public static final ChronoUnit INACTIVE_UNIT = ChronoUnit.HOURS;
+    public static final long INACTIVE_TEST_INTERVAL = 15;
+    public static final ChronoUnit INACTIVE_TEST_UNIT = ChronoUnit.MINUTES;
```

Comment:
While at it, might also want to make this configurable via config?

Reply:
i can but that should probably wait until the config template update is completed
since those settings should be be needed to get manually configured by any user every time they set up the bot

should i add a //TODO?

Reply:
added